modules/apps/object/object-test/src/testIntegration/resources/com/liferay/object/web/internal/portlet/action/test/dependencies/test-object-definition-state-manager.json

```diff
@@ -0,0 +1,246 @@
+{
+	"accountEntryRestricted": false,
+	"accountEntryRestrictedObjectFieldName": "",
+	"actions": {
```

Comment:
Hi @victorKammerer , I believe this document will be restructured soon, that's why we're stuck for now. I'm closing this PR and will be analyzing it through the liferay-object repo. Thank you and congrats for your first analysis! 
![image](https://github.com/rijema/liferay-portal/assets/41641596/da888167-8305-4d4d-ba6f-df2c5910129d)